notebooks/psf_photometry/NIRCam_PSF_Photometry_Example.ipynb

```diff
@@ -2679,7 +2679,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.11.7"
+   "version": "3.8.10"
```

Comment:
Why has this changed to version 3.8.10?

Reply:
This is only for the deployment. So no review is needed.